packages/fxa-shared/lib/logger.ts

```diff
@@ -1,19 +1,15 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { ILogger } from '../log';
```

Comment:
nit: I'm not sure if there is a lot of added value in having a separate file for `ILogger`, possibly if we had plans it use it somewhere else?